src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -234,6 +234,15 @@ private static IEnumerable<ISetupInstance> EnumerateVisualStudioInstances()
         private static ISetupInstance LocateVisualStudioInstance(ImmutableHashSet<string> requiredPackageIds)
         {
             var vsInstallDir = Environment.GetEnvironmentVariable("VSInstallDir");
+            if (vsInstallDir is null)
+            {
+                vsInstallDir = Environment.GetEnvironmentVariable("__UNITTESTEXPLORER_VSINSTALLPATH__");
```

Comment:
😕 When I tried to port this to tunnelvisionlabs/MouseFastScroll, I found this environment variable was not set. @ManishJayaswal What's the difference between `__UNITTESTEXPLORER_VSINSTALLPATH__` and `VSAPPIDDIR`, and why should I pick one over the other?

Reply:
hmmm I don't know. @AbhitejJohn - do you know which one should be used?